hw1_iphone/index.html

```diff
@@ -12,6 +12,121 @@
     />
   </head>
   <body>
-    <p>hw</p>
+    <div class="container">
+      <div class="iphone">
+        <div class="phone-box">
+          <div class="tools">
+            <div class="front-camera_top"></div>
+            <div class="front-camera_left"></div>
+            <div class="microphone"></div>
+          </div>
+          <div class="screen">
+            <div class="top-bar">
+              <div class="signal-icons">
+                <img
+                  class="signal-round1"
+                  src="/hw1_iphone/images/Oval.png"
+                  alt="image"
+                />
+                <img
+                  class="signal-round2"
+                  src="/hw1_iphone/images/Oval.png"
```

Comment:
In future it would be better to use svg images for such goal

Reply:
next time would use the .svd format